frontend/src/mixins/OrgUnitValidity.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  computed: {
+    orgUnitValidity () {
+      if (this.entry.org_unit || this.entry.validity) {
+        if (this.entry.org_unit.validity < this.entry.validity) {
```

Comment:
You're comparing two objects by value here; I don't think that works the way you intend it to:

```javascript
> a = {from: 0, to: 42}
{ from: 0, to: 42 }
> b = {from: -100, to: 100}
{ from: -100, to: 100 }
> a < b
false
> b < a
false
> b == a
false
> a <= b
true
> b <= a
true
```